pkg/multimodelconfig/configmap.go

```diff
@@ -0,0 +1,110 @@
+package multimodelconfig
```

Comment:
As I understand it, this isn't only for multimodel and in the degenerate case, we will use the same mechanism for a single model. What do you think about putting this code in the TrainedModel api package?

Reply:
We prefer to put configmap in pkg/multimodelconfig but not in TrainedModel api package since we it is used internally and we don't want to expose this package externally.

Reply:
If this is used for both multimodel and single model, what do you think about
```
/pkg/modelconfig
```